adaptors/hybrisalsadaptor/hybrisalsadaptor.cpp

```diff
@@ -55,7 +55,7 @@ void HybrisAlsAdaptor::stopSensor()
 void HybrisAlsAdaptor::processSample(const sensors_event_t& data)
 {
     TimedUnsigned *d = buffer->nextSlot();
-    d->timestamp_ = quint64(data.timestamp * .0001);
+    d->timestamp_ = quint64(data.timestamp * .001);
```

Comment:
Because it always has been this way in sensorfw, and it's not going to change any time soon just to appease android/hybris.